File: ui/src/types/item-export-details.ts
export interface ItemExportDetails {
  id: number;
  itemCode: string;
  countryOfOrigin?: string;
  harmonizedSystemCode?: string;
  exportLicenseRequired?: boolean;
  eccn?: string; 
  itemId: number;
  itemDescriptionForExports: string;
  exportProductGroupCode: string;
  exportProductGroupName: string;
  depbRateListSrlNo: string;
  depbRate?: string;
  depbValueCap?: string;
  depbRemarks: string;
  dutyDrawbackSrlNo: string;
  dutyDrawbackRate?: string;
  dutyDrawbackRateType: string;
  dutyDrawbackValueCap?: string;
  dutyDrawbackRemarks: string;
}

export type CreateItemExportDetailsData = Omit<ItemExportDetails, 'id'>;

export type UpdateItemExportDetailsData = Partial<CreateItemExportDetailsData>;

File: api/Core/Model/ItemExportDetails.cs
using System;

namespace Xcianify.Core.Model
{
    public class ItemExportDetails : BaseModel
    {
        public int Id { get; set; }
        public int ItemId { get; set; }
        public string ItemDescriptionForExports { get; set; }
        public string ExportProductGroupCode { get; set; }
        public string ExportProductGroupName { get; set; }
        public string DepbRateListSrlNo { get; set; }
        public decimal? DepbRate { get; set; }
        public decimal? DepbValueCap { get; set; }
        public string DepbRemarks { get; set; }
         public string DutyDrawbackSrlNo { get; set; }
        public decimal? DutyDrawbackRate { get; set; }
        public string DutyDrawbackRateType { get; set; }
        public decimal? DutyDrawbackValueCap { get; set; }
        public string DutyDrawbackRemarks { get; set; }
        public bool IsDeleted { get; set; }
        public bool IsDeleted { get; set; }
    }
}
